### ISSUE TITLE
LSTM fit function takes 30% more time with tensorflow 2.4 than version 2.2

### ISSUE THREAD

**Issue by DoruSonic** [rstudio/tensorflow#460]
I'm using keras (LSTM neural network) to create a model that predicts future values and I was also monitoring both the RMSE and time taken. I've noticed that after upgrading the tensorflow version that it took more time to finish.
In the 2.2 version it takes around 31 minutes and with the same hyperparameters on the 2.4 version it takes 43 minutes. I run it a couple of times in both versions and these results were consistent. With other hyperpameters it went from taking 146 minutes to 232 minutes.

Note: I have the correct cuda and cudnn drivers installed and the dlls are loaded successfully. Using R 4.0.5 ; Rstudio 1.4.1103 ; Keras 2.4.0

**Comment by t-kalinowski** [comment]
Hi, thanks for filing!
This issue doesn’t seem specific to the R interface to tensorflow, but rather the underlying tensorflow package. Please take a look through the issues [here](https://github.com/tensorflow/tensorflow/issues?q=label%3Atype%3Aperformance+label%3A%22TF+2.4%22) to see if there is already a discussion, and if not and consider filing a new issue there.

If I’m mistaken and this is specific to the R interface, please feel free to reopen this issue.
